examples/platform/nrf528xx/app/project_include/OpenThreadConfig.h

```diff
@@ -49,6 +49,8 @@
 #define OPENTHREAD_CONFIG_NCP_UART_ENABLE 1
 #define OPENTHREAD_CONFIG_IP6_SLAAC_ENABLE 1
 
+#define OPENTHREAD_CONFIG_NUM_MESSAGE_BUFFERS 32 // Default is 160, set to 32
```

Comment:
supernit: 'set to 32' seems redundant information. Also wondering if default may change or not.

Maybe something like `// ensure specific size for this chipset: overly large numbers consume too many resources`

Reply:
I suggest use OpenThread's default value for this.
```suggestion
#define OPENTHREAD_CONFIG_NUM_MESSAGE_BUFFERS 44
```